WalletWasabi.Gui/Dialogs/AvaloniaControlsExtensions.cs

```diff
@@ -12,18 +12,18 @@ namespace Avalonia.Controls
 {
 	public static class AvaloniaControlsExtensions
 	{
-		public static async Task<string> ShowAsync(this SaveFileDialog me, Window parent, bool fallBack)
+		public static async Task<string> ShowAsync(this SaveFileDialog me, Window? parent, bool fallBack)
```

Comment:
Can the parent of a dialog be null? Is that legit?

Reply:
I don't think so, you need a window on which you show the dialog. I would revert changes in this file.